src/test/java/cat/udl/eps/softarch/mypadel/steps/CreateMatchStepDefs.java

```diff
@@ -29,6 +32,8 @@
 
     private GregorianCalendar cancelationDeadline = new GregorianCalendar(2017, 9, 30);
 
+    private DateTimeFormatter dateTime = DateTimeFormatter.ISO_DATE_TIME;
+
     @When("^I create a new public match$")
```

Comment:
Consider adding some of the attributes as step parameters to make the step more reusable instead of fixing the duration, court type, level,...